cmd/buildah/images.go

```diff
@@ -268,6 +274,13 @@ func outputImages(ctx context.Context, images []storage.Image, format string, st
 			outputUsingFormatString(truncate, digests, params)
 		}
 	}
+
+	if !found {
```

Comment:
Nit you can combine these args if statements into a single if

if !found && argName != ""

Reply:
Thanks, updated, and I changed the related test file.